src/main/java/org/onebusaway/alexa/lib/ObaUserClient.java

```diff
@@ -90,6 +90,21 @@ public ObaUserClient(String obaBaseUrl) throws URISyntaxException {
     }
 
     /**
+     * Returns details about a particular stop, given it's stopId
+     * @param stopId
+     * @return details about a particular stop, given it's stopId
+     */
+    public Optional<ObaStopResponse> getStopDetails(String stopId) {
+        ObaStopResponse response = new ObaStopRequest.Builder(stopId).build().call();
+        log.debug("ObaStopRequest returned " + response.toString());
+        if (response.getCode() == ObaApi.OBA_OK) {
```

Comment:
When might the response be anything but OBA_OK? If only when the OBA API is down, then I'd rather throw an exception here than force the client to unwrap the Optional.